.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,31 @@
+---
+blank_issues_enabled: false
+contact_links:
+  -
+    about: Visit our Discussions page
+    name: 💡 Feature requests
+    url: https://github.com/meteor/meteor/discussions
+  -
+    about: Start a discussion on the Meteor forums
```

Comment:
We should avoid the word discussion here, too confusing.

Reply:
Fixed